### ISSUE TITLE
[NEW FEATURE] Support homebrew

### ISSUE THREAD

**Issue by zJiaJun** [xfangfang/Macast#56]
## New feature
Support homebrew
i like this app.

## Additional context
you konw,homebrew is popular on the MAC

**Comment by xfangfang** [comment]
Yes, upload to each package management tool is working in progress. However, there are still some bugs in the program. Due to the lack of time, the priority to solve the bugs of the program is higher.

I will support homebrew in the future, thanks for your advice.

**Comment by cocoonkid** [comment]
+1

**Comment by zhangcheng** [comment]
Just saw @moonfruit already made it happened (Homebrew/homebrew-cask#120299).

And also saw that he [mentioned](https://github.com/Homebrew/homebrew-cask/pull/120299#discussion_r824327230) the version tag system being unusual (not [semver](https://semver.org/) compatible I guess), which needs @xfangfang to consider.

**Comment by xfangfang** [comment]
Great!!! I don't even know about this.
 I will solve this problem as soon as possible.

**Comment by moonfruit** [comment]
Yes, I just add this to homebrew-cask.
